packages/front-end/components/Experiment/SinglePage.tsx

```diff
@@ -511,6 +558,80 @@ export default function SinglePage({
           mutate={mutate}
         />
       )}
+      {showStartExperiment && experiment.status === "draft" && (
+        <Modal
+          open={true}
+          size="md"
+          closeCta={
+            hasVerifiedConnection ? (
+              <button
+                className="btn btn-primary"
+                onClick={() => setShowStartExperiment(false)}
+              >
+                Close
+              </button>
+            ) : (
+              // This is a bit odd, but design requested we use the closeCTA as an override in this case
```

Comment:
There is a related ticket here for it - https://www.notion.so/growthbook/UI-Issues-to-Work-into-Releases-14ed60043f2f4b1cae74db9ec57afec8?p=c31d55afe8424e04827092c249d10941&pm=s